sdk/servicebus/service-bus/test/batchReceiver.spec.ts

```diff
@@ -1215,7 +1208,7 @@ describe("Batching - disconnects", function(): void {
       await receiver.receiveBatch(10, { maxWaitTimeInMs: 1000 });
       throw new Error(testFailureMessage);
     } catch (err) {
-      err.message.should.not.equal(testFailureMessage);
+      err.message && err.message.should.not.equal(testFailureMessage);
```

Comment:
In the browser, the error is 
`LOG: CloseEvent{isTrusted: true}` when logged.

This is to verify the error message only when it exists.

Reply:
You're saying that in the browser, what gets thrown is a string? Is this only when running headless chrome or in general?

Reply:
Yes, tested headless chrome only. Will verify other browsers.

Reply:
Same error in Chrome
![image](https://user-images.githubusercontent.com/10452642/85323196-729c1f00-b47c-11ea-97b8-5996449031cd.png)